drivers/timer/cc13x2_cc26x2_rtc_timer.c

```diff
@@ -190,10 +190,10 @@ int z_clock_driver_init(struct device *device)
 	initDevice();
 	startDevice();
 
-	/* Enable RTC interrupt. */
-	IRQ_CONNECT(DT_INST_0_TI_CC13XX_CC26XX_RTC_IRQ_0,
-		DT_INST_0_TI_CC13XX_CC26XX_RTC_IRQ_0_PRIORITY,
-		rtc_isr, 0, 0);
+//	/* Enable RTC interrupt. */
```

Comment:
BLE requires fairly tight coupling with the RTC. I've connected the rtc irq in lll.c and simply call the rtc_isr() manually when channel 0 is triggered. A better solution is needed in the long run.

Reply:
Thinking aloud here, Add a Kconfig that conditionally compiles the old code and new code as required

Reply:
Covered under another task.